packages/grid-html-export/src/components/StandardTableHtmlCopyToClipboardButton.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
+import { useCallback, useState } from "react";
+import { StandardTableProps } from "@stenajs-webui/grid";
+import { FlatButton, FlatButtonProps } from "@stenajs-webui/elements";
+import * as clipboard from "clipboard-polyfill";
+import { CustomCellFormatters } from "../transformers/ConfigTransformer";
+import { renderHtmlForStandardTable } from "../util/HtmlRenderer";
+
+interface StandardTableHtmlExportButtonProps<
+  TItem,
+  TColumnKey extends string,
+  TColumnGroupKey extends string
+> extends Pick<
+      StandardTableProps<TItem, TColumnKey, TColumnGroupKey>,
+      "config" | "items"
+    >,
+    Pick<FlatButtonProps, "size"> {
+  formatters?: CustomCellFormatters<TItem, TColumnKey>;
+  renderContent?: (content: string) => string | null;
+  label?: string;
+  labelAfterCopy?: string;
+}
+
+export const StandardTableHtmlCopyToClipboardButton = function StandardTableHtmlExportButton<
+  TItem,
+  TColumnKey extends string,
+  TColumnGroupKey extends string
+>({
+  config,
+  renderContent,
+  size,
+  items,
+  formatters,
+  label = "Copy to clipboard",
+  labelAfterCopy = "Content copied!",
+}: StandardTableHtmlExportButtonProps<TItem, TColumnKey, TColumnGroupKey>) {
+  const [contentCopied, setContentCopied] = useState<boolean>(false);
+  const onClickExportHtml = useCallback(async () => {
```

Comment:
Export the functionality in this callback to function in another file.
So that we can reuse it in other components if necessary.

Not `setContentCopied(true)` though, that is specific to this component.